pex/bin/pex.py

```diff
@@ -519,8 +519,11 @@ def to_python_interpreter(full_path_or_basename):
     # affect usages of the interpreter(s) specified by the "--python" command line flag.
     constraints = options.interpreter_constraint
     validate_constraints(constraints)
-    rc_variables = Variables.from_rc(rc=options.rc_file)
-    pex_python_path = rc_variables.get('PEX_PYTHON_PATH', '')
+    if not ENV.PEX_IGNORE_RCFILES:
+      rc_variables = Variables.from_rc(rc=options.rc_file)
```

Comment:
The options.rc_file value is `None` by default. To be non-`None` pex had to be invoked from the cli with `--rcfile=[path]`. As such you're setting the precedent rule here of env beats flags. That does not seem kosher. Is that what you're intending here or have I read things wrong?

Reply:
Honestly, the flag really makes no sense. It shouldn't exist at all fwict. I'd be fine with considering this not your problem. I've filled https://github.com/pantsbuild/pex/issues/674

Reply:
Even if the option is `None`, `Variable.from_rc` will try to load stuff from well-known files (i.e. `/etc/pexrc`, `~/.pexrc`...), which leaves us with no way of specifying here "I actually want you to ignore all pexrc files".

Reply:
Well, the condition could be `if options.rc_file or not ENV.PEX_IGNORE_RCFILES`.

Reply:
So I'd still like a way of expressing "Please do not try to load pexrc files", so would you be okay with something like "If the flag is None, and the env variable PEX_IGNORE_RCFILES is set, please do not load any pexrc file? That would create slightly inconsistent semantics for the env variable, until the flag is removed.

Reply:
Yes. I think we're saying the same thing. SGTM.